### PR TITLE
update ruby version to 2.7.2

### PR DESCRIPTION
**Pull request by jeroig**
#### Description:

This update attempt to remove this warning

````
warning: parser/current is loading parser/ruby27, which recognizes
warning: 2.7.2-compliant syntax, but you are running 2.7.1.
````

Also was needed to update ffi gem in order to [resolve type 'size_t' (Big Sur - M1)](https://stackoverflow.com/questions/66104935/rails-typeerror-unable-to-resolve-type-size-t-big-sur-m1)

### CODE REVIEW

**Review by santib — APPROVED** [review]

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.7.1
+2.7.2
```
Comment (sebaherrera07) [inline .ruby-version:1]: Maybe you need to also update the version in `circleci/config.yml`

Comment (santib) [reply .ruby-version:1]: I think Seba is right, and also in the `Gemfile`, `Gemfile.lock` (by running bundle), and `Dockerfile`

Comment (jeroig) [reply .ruby-version:1]: I totally agree !!

**Comment by jeroig** [comment]
If anyone can help me why `circleci` failed I appreciate because I don't have experience over that

**Comment by jeroig** [comment]
I decided to close this PR because circleci fail .

And probably is because this PR come from a fork project (not a branch from itself)
For more info [see](https://github.com/rootstrap/rails_api_base/issues/264)

The same change will be sending since other PR

**Pull request closed without merging**